aio/content/marketing/resources.json

```
@@ -282,6 +282,12 @@
             "title": "Scully",
             "logo": "https://raw.githubusercontent.com/scullyio/scully/main/assets/logos/PNG/Green/scullyio-logo-green.png",
             "url": "https://scully.io"
+          },
+          "ng-mocks": {
+            "desc": "A testing library for creating mock services, components, directives, pipes and modules out of annoying dependencies in unit tests.",
+            "title": "ng-mocks",
+            "logo": "",
+            "url": "https://github.com/ike18t/ng-mocks"
```

FYI, you could append `#readme` to the URL and have the page scroll automatically to the README when visiting:

```suggestion
            "url": "https://github.com/ike18t/ng-mocks#readme"
```

(This is just fyi - it's up to you what URL to use.)

---

done